act-rules-format.bs

```diff
@@ -106,45 +106,68 @@ An ACT Rule MUST have a description that is in plain language, and provides a br
+An ACT Rule MUST list an identifier and reference for each related reference that it is written to directly test. When related references are listed, the rule MUST indicate that these are not required for conformance. This MAY be omitted when rules are designed to be part of a [=composite rule=].
 
 <div class=example>
-  <p><strong>Example</strong>: The outcome of this rule maps to the accessibility requirements in the following manner:</p>
+  <p><strong>Example</strong>: The outcome of this rule indicates conformance beyond the related WCAG 2.1 success criteria:</p>
   <ul>
-    <li>`failed`: The success criteria are *not satisfied*</li>
-    <li>`passed` or `inapplicable`: Further testing is necessary to determine if the success criteria are satisfied</li>
+    <li>[G127: Identifying a Web page's relationship to a larger collection of Web pages](https://www.w3.org/TR/WCAG20-TECHS/G127.html)</li>
+  </ul>
+</div>
+
+
+Rule Mapping {#rule-mapping}
+----------------------------
+
+For all [accessibility requirements](#accessibility-requirements) and [related references](#related-references) listed, the ACT Rule MUST indicate how each outcome relates to the listed accessibility references. That is, the rule specifies if each accessibility reference is `satisfied`, `not satisfied`, or `undetermined` for the rule outcomes `passed`, `failed`, and `inapplicable`.
+
+Every mapping MUST be consistent with the related accessibility references. When rules refer to accessibility requirements, the mapping MUST result in either `satisfied` or `not satisfied` for any of the rule outcomes. That is, the mapping is not `undetermined` for all outcomes when mapping to accessibility requirements. Such mappings indicate related references rather than requirements.
+
+<div class=example>
+  <p><strong>Example</strong>: The outcome of this rule maps to the listed accessibility requirements in the following manner:</p>
+  <ul>
+    <li>`failed`: *not satisfied* (the success criteria are not met)</li>
+    <li>`passed` or `inapplicable`: *undetermined* (further testing is necessary to determine if the success criteria are satisfied)</li>
   </ul>
 </div>
 
 <div class=note>
   **Note**: In WCAG 2.1, success criteria do not evaluate to `passed`, `failed` or `inapplicable`. Rather they can be *satisfied* (or not). (See the [WCAG 2.1 definition: satisfies a success criterion](https://www.w3.org/TR/WCAG21/#dfn-satisfies).) If a success criterion is *not satisfied* a web page can only conform if there is a conforming alternative version, as described in [WCAG 2.1 Conformance Requirement 1](https://www.w3.org/TR/WCAG21/#cc1).
 </div>
 
+<div class=note>
+  **Note**: ACT Rules are commonly partial tests for accessibility references. That is, the outcomes `passed`, `failed`, and `inapplicable` map to `undetermined`. A rule is a complete test when the outcomes only map to `satisfied` and `not satisfied`.
```

Comment:
That would mean in WCAG you can't have rules that are a "complete test". A HTML page title rule is inapplicable for PDF. Perhaps if passed and failed map to satisfied and not satisfied? I'm not sure that works.

Reply:
If the rule only checks HTML, then it is by definition not a complete test for WCAG success criteria, which are technology-agnostic.